src/main/resources/application.yml

```diff
@@ -32,7 +32,7 @@ spring:
 
     hikari:
       pool-name: OffenderCaseNotes-CP
-      connectionTimeout: 1000
+      connectionTimeout: 5000
```

Comment:
since we know after warm up the first call still takes about 2 seconds, need to factor that into any timeouts